### PR TITLE
React form controls

### PR DESCRIPTION
**Pull request by OSAdams**
image

### CODE REVIEW

File: react-form-controls/src/registration-form.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  handleSubmit(event) {
+    console.log('state: ', this.state);
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+      <form onSubmit={ this.handleSubmit }>
+        <label htmlFor="username">Username:</label>
```
Comment (scbowler) [inline react-form-controls/src/registration-form.jsx:31]: The `htmlFor` prop should match the `id` prop of the associated `input`. So if you are using `htmlFor` on the label the `input` should have a matching `id`.

File: react-form-controls/src/registration-form.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <input type="text" name="username" placeholder="JonSnow" value={ this.state.username } onChange={ this.handleUsernameChange } />
```
Comment (scbowler) [inline react-form-controls/src/registration-form.jsx:32]: You know nothing Jon Snow

**Review by scbowler — APPROVED** [review]
Looks good! 👍 

1 FYI comment, no changes required.

File: react-form-controls/src/registration-form.jsx (hunk 2 above)
Comment (OSAdams) [reply react-form-controls/src/registration-form.jsx:32]: image

**Pull request merged**